src/activities/dto/create-activity.dto.ts

```diff
@@ -1,35 +1,44 @@
 import { ApiProperty } from '@nestjs/swagger';
 
 export class CreateActivityDto {
-  @ApiProperty({ description: 'name', type: 'string', example: 'testname' })
-  readonly name: string;
+    @ApiProperty({description: 'name',
+    type: 'string',
+    example: 'testname',})
+    readonly name: string;
 
-  @ApiProperty({
-    description: 'destination_id',
+    @ApiProperty({description: 'destination_id',
     type: 'string',
```

Comment:
I'm confused. Shouldn't this be a an Numeric value rather than string. Something like BigInt.

Correct me if I'm wrong here.

Reply:
It should be number